.nvmrc

```diff
@@ -1 +1 @@
-24.12.0
```

Comment:
This version breaks in bazel 8.5
https://github.com/bazel-contrib/rules_nodejs/issues/3887